en/tokovoip.md

```diff
@@ -0,0 +1,155 @@
+
+To get the IP:
+![](https://screensaver01.zap-hosting.com/index.php/s/2F4szBejmwFsF5T/preview)
+To get the port: 
+![](https://screensaver01.zap-hosting.com/index.php/s/344i2m8aqjXf3fD/preview)
+
+> Note: Customers who have booked a 'Own IPV4 Address' will still need to specify the **FTP-Browser** IP address as shown above. 
+
+For the **TSChannel** and **TSPassword** you will set these to the names of the channel name/password you have defined on your Teamspeak Server. If you have not created this, you will need to create the channel. 
+
+> This will be the channel that your players are moved into for communications once they have fully loaded into your FiveM server. Therefore, a password is recommended so any players that are not on the server cannot join the channel. 
+
+In our example we configure it as:  
+![](https://screensaver01.zap-hosting.com/index.php/s/Ce56EYFADjt7cWn/preview)
+
+For the **TSChannelWait** you will put the **exact** waiting channel name your players will enter when loading into the server. You will need to create this channel, and can name it anything you wish. 
+
+> IMPORTANT: You **must** have  **TokoVOIP** somewhere in the name of this channel. 
+
+In our example, we configure it as shown:
+
+![](https://screensaver01.zap-hosting.com/index.php/s/RWbLjncit6ttT2g/preview)
+
+Then save your changes, and ensure they upload successfully back to the server.
+
+### Configure server.cfg
+
+Once you have configured both files, you will need to then modify your `server.cfg` file.
+
+You can do this either via FTP, or directly through the ZAP 'Configs' tab. 
+
+Once you have opened your server.cfg, you will need to add the following start lines in your server.cfg:
+
+```
+start ws_server
+start tokovoip_script
+```
+![](https://screensaver01.zap-hosting.com/index.php/s/mqr2E42jSAYHtMz/preview)
+
+### Test the plugin
+
+Once you have configured both the ws_server and tokovoip_script and have added the start lines to the server.cfg, you have finished the base install of TokoVOIP. 
+
+Then, **join your teamspeak waiting channel** that you have created, **start the server**, and **test it directly.
+
+![](https://screensaver01.zap-hosting.com/index.php/s/9yiZP2NgyJ7DyMS/preview)
+
+## 📃 Extra Configurations
+
+TokoVOIP does support further configurations for things such as Radios, setting which key does what, and much more.
+
+We advise you check all available config files and configure them to your liking.
+
+For specific information relating to the configurations, you should visit their [Github](https://github.com/Itokoyamato/TokoVOIP_TS3)
+
+> ZAP is unable to assist you with advanced configurations for TokoVOIP. 
+
+
+## ❓ Problems? 
+
+If you encounter any issue, its best to review your server console for any errors and ensure you have defined the correct IP/ports within your configs.
+
+> Note: Customers who have booked a 'Own IPV4 Address' will still need to specify the **FTP-Browser** IP address as shown in the guide. 
```

Comment:
this would probably be better suited to be put near the explenation for the configuration of it, instead of down here